src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -86,6 +101,14 @@
             <tag name="serializer.encoder" />
         </service>
 
+        <service id="serializer.encoder.yaml" class="Symfony\Component\Serializer\Encoder\YamlEncoder">
```

Comment:
This is broken, `symfony/yaml` may not be installed and we now have an error when trying to use the serializer... See https://travis-ci.org/nelmio/NelmioApiDocBundle/jobs/305754242.

Reply:
Hmm. My bad. I'll look at this soon. Thanks for the report!

Reply:
Thanks!

Reply:
See https://github.com/symfony/symfony/pull/25179